src/libutil/typedesc.cpp

```diff
@@ -175,8 +175,6 @@ TypeDesc::c_str () const
     std::string result;
     if (aggregate == SCALAR)
         result = basetype_name[basetype];
-    else if (aggregate == MATRIX44 && basetype == FLOAT)
-        result = "matrix";
```

Comment:
You know, now that I'm looking at TypeDesc::from_string versus TypeDesc::c_str(), I'm seeing some inconsistencies. Let me take a closer look at this issue and see what can be changed without breaking much in OSL. Give me a little time and I'll report back here. We may be ripe for some cleanup, so I'm taking a hard line on anything quite yet.